Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1364,6 +1364,19 @@ theorem small_iff_lift_mk_lt_univ {α : Type u} :
   · rintro ⟨c, hc⟩
     exact ⟨⟨c.out, lift_mk_eq.{u, _, v + 1}.1 (hc.trans (congr rfl c.mk_out.symm))⟩⟩
 
+lemma zero_lt_ord_iff (c : Cardinal) : 0 < c.ord ↔ c ≠ 0 := by
```

Comment:
It looks like you can golf with [Cardinal.lt_ord](https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Ordinal/Basic.html#Cardinal.lt_ord), and then `Ordinal.eq_zero_or_pos`.

Reply:
Thanks very much!